crate/server/src/core/cover_crypt/rekey_keys.rs

```diff
@@ -297,12 +304,12 @@ async fn update_all_active_usk(
     // Refresh the User Decryption Key that were found
     if let Some(unique_identifiers) = &locate_response {
         //instantiate a CoverCrypt User Key Handler
-        let handler = UserDecryptionKeysHandler::instantiate(cover_crypt, &msk_obj.1)?;
+        let handler = &mut UserDecryptionKeysHandler::instantiate(cover_crypt, &msk_obj.1)?;
```

Comment:
The MSK is modified here and should be saved back to the DB. It is not necessary for now since we do not support modifying the tracing level. But this will be a bug the day we do, if we do. I would suggest saving the MSK.